wings/1.0/configuration.md

```diff
@@ -29,3 +30,69 @@ docker:
       username: "registryusername"
       password: "registrypassword"
 ```
+
+## Custom Network Interfaces
+
+You can change the network interface that Wings uses for all containers by editing the network name; it is by default set to `pterodactyl_nw`. For example, to enable Docker host mode change the network name to `host`.
+
+::: warning
+Changing network mode to `host` grants Pterodactyl direct access to all machine interfaces and Panel users can bind to any IP or Port even if it's not allocated to their container. You will lose all benefits of Docker network isolation. It is not recommended for public installations that are hosting other users' servers.
+:::
+
+### Example of usage
+
+```yml
+docker:
+  network:
+    name: host
+```
+
+After making changes, the following commands will stop the Wings, remove the Pterodactyl network, and start the Wings again. Run at your own risk.
+`systemctl stop wings && docker network rm pterodactyl_nw && systemctl start wings`
+
+## Enabling Cloudflare proxy
+
+Cloudflare proxying of the Wings isn't beneficial since users will be connecting to the machine directly and bypassing any Cloudflare protection. As such, your Node machine IP will still be exposed.
+
+To enable Cloudflare proxy, you must change the Wings port to one of the Cloudflare HTTPS ports, such as 8443, because Cloudflare only supports HTTP on port 8080. Select your Node in the Admin Panel, and on the settings tab, change the port.
+
+You are unable to proxy the SFTP port through Cloudflare unless you have their enterprise plan.
```

Comment:
That doesn't seem soo fare. Explain!

Reply:
I'm not sure what do you mean by that comment

Reply:
Why only enterprise plan!?!?!?!

Reply:
You used to need enterprise plan but with CF Spectrum being publically available it's significantly cheaper (although you still need to pay for the proxied bandwidth)

Reply:
That is not our decision, that is Cloudflare's.  You can only proxy raw TCP and UDP traffic using Cloudflare if you have Spectrum or Magic Transit (which are available for additional cost if you have their enterprise plan).

Reply:
I guess I'll  just have to spend around $800 per month then! :)